addons/mail_group/views/portal_templates.xml

```diff
@@ -273,50 +197,122 @@
                 <button class="btn btn-link o_mg_read_more"
                     t-attf-data-href="/groups/#{slug(group)}/#{slug(parent_message)}/get_replies"
                     t-attf-data-last-displayed-id="#{messages[-1].id}">
-                    show <t t-esc="msg_more_count"/> more replies
+                    <t t-esc="msg_more_count"/> more replies
                 </button>
             </p>
         </div>
     </template>
 
+    <template id="message_attachments">
+        <div class="o_mg_link_parent">
+            <t t-set="attachments" t-value="message.sudo().attachment_ids"/>
+            <t t-set="access_toknes" t-value="attachments.generate_access_token()"/>
```

Comment:
Typo + would remove from template, as their duty is not to have an effect on models (generate access token modifies the record).

Reply:
Hum, I'll remove this and generate the access token on the controller side